### PR TITLE
feature/LVJAV442-43

### PR DESCRIPTION
**Pull request by martaprotsak**
Add FileController

### CODE REVIEW

File: src/main/java/com/softserve/ita/java442/cityDonut/constant/ErrorMessage.java [hunk 1]
```diff
@@ -4,4 +4,10 @@
     public static final String USER_NOT_FOUND_BY_ID = "The user does not exist by this id: ";
     public static final String PROJECT_NOT_FOUND_BY_ID = "The project does not exist by this id";
     public static final String INCORRECT_USER_PASSWORD = "The user`s password is incorrect: ";
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/constant/ErrorMessage.java:7]: delete row

File: src/main/java/com/softserve/ita/java442/cityDonut/exception/CustomExceptionHandler.java [hunk 2]
```diff
@@ -40,6 +41,18 @@ public final ResponseEntity notFoundException(WebRequest request){
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
     }
 
+    @ExceptionHandler(MalformedURLException.class)
+    public final ResponseEntity malformedURLException(WebRequest request){
+        ExceptionResponse exceptionResponse = new ExceptionResponse(getErrorAttributes(request));
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
+    }
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/exception/CustomExceptionHandler.java:61]: 47, 53 rows. Are u sure that HttpStatus should be Not_Found?

File: src/main/java/com/softserve/ita/java442/cityDonut/constant/ErrorMessage.java (hunk 1 above)
Comment (martaprotsak) [reply src/main/java/com/softserve/ita/java442/cityDonut/constant/ErrorMessage.java:7]: It's more convenient to read messages by blocks. For example, I have in block down all messages needed for files, so if I need to add a new one, I will check all messages in this block and then decide what to do. If all messages are in one block, I will have to read many of them. If you whant, you can check code convention of this file in GreenCity

File: src/main/java/com/softserve/ita/java442/cityDonut/exception/CustomExceptionHandler.java (hunk 2 above)
Comment (martaprotsak) [reply src/main/java/com/softserve/ita/java442/cityDonut/exception/CustomExceptionHandler.java:61]: Which one do you propose instead of NOT_FOUND?

Comment (oprokipchuk) [reply src/main/java/com/softserve/ita/java442/cityDonut/exception/CustomExceptionHandler.java:61]: BAD_REQUEST

**Review by NazarRyvak — APPROVED** [review]

**Review by oprokipchuk — APPROVED** [review]

**Review by martavoloshyn — APPROVED** [review]

**Pull request merged**